provision/docker/containers.go

```diff
@@ -308,13 +308,18 @@ func (p *dockerProvisioner) runCommandInContainer(image string, command string,
 	if stderr == nil {
 		stderr = ioutil.Discard
 	}
+	var envs []string
+	for _, e := range app.Envs() {
```

Comment:
What about using `provision.EnvsForApp()`? This way dynamic vars like TSURU_HOST and PORT will also be injected.